ovos_plugin_common_play/ocp/utils.py

```diff
@@ -26,7 +26,7 @@ def find_mime(uri):
 
 def available_extractors():
     return ["/", "http"] + \
```

Comment:
okay, I missed that paths without the leading `file://` are now properly formatted and handled independently. I will revert with some comments around this to avoid confusion later